pkg/deploy/testdata/happyPathDeploymentFileBlueGreen.yaml

```diff
@@ -0,0 +1,34 @@
+version: v1
+kind: kubernetes
+application: app
+# Map of Deployment Targets
+targets:
+  # This in the name of a deployment, underneath it are its configuration
+  test1:
+    account: account1
+    namespace: dev
+    strategy: batman
+    # This is a list of manifest names that should be included in this deployment
+# The list of manifests sources
+manifests:
+  # This specifies a specific manifest file
+  - path: testdata/testManifest1.yaml
+    targets: ["test1"]
+# The map of strategies, a deployment target will reference one of these
+strategies:
+  # this is the name for the strategy
+  batman:
+    # This map key, is the deployment strategy type
+    blue-green:
+      shutdownOldVersionAfter:
+        steps:
```

Comment:
I don't think it should be nested as "steps" here because they happen in parallel

It should be consistent with the multi-env `beforeDeployment` constraints ([see PRD](https://armory.slab.com/posts/multi-environment-deployment-prd-da5btmqc#hio9r-yaml)), which also happen in parallel.